### ISSUE TITLE
Egregious traceback for failed jobs

### ISSUE THREAD

**Issue by mesmith75** [ganga-devs/ganga#951]
We get nasty tracebacks for failed jobs that have timed out / been killed by the WN and so don't have anything in the output sandbox, eg
```python
ERROR    Exception raised executing 'job_finalisation' in Thread 'Job 1466.16 Finalizing':
Traceback (most recent call last):
  File "/afs/cern.ch/user/m/masmith/cmtuser/GANGA/GANGA_HEAD/install/ganga/python/Ganga/Core/GangaThread/WorkerThreads/WorkerThreadPool.py", line 114, in __worker_thread
    result = item.command_input.function(*these_args, **item.command_input.kwargs)
  File "/afs/cern.ch/user/m/masmith/cmtuser/GANGA/GANGA_HEAD/install/ganga/python/GangaDirac/Lib/Backends/DiracBase.py", line 773, in job_finalisation
    DiracBase._internal_job_finalisation(job, updated_dirac_status)
  File "/afs/cern.ch/user/m/masmith/cmtuser/GANGA/GANGA_HEAD/install/ganga/python/GangaDirac/Lib/Backends/DiracBase.py", line 747, in _internal_job_finalisation
    execute("getOutputSandbox(%d,'%s')" % (job.backend.id, job.getOutputWorkspace().getPath()), cred_req=job.backend.credential_requirements)
  File "/afs/cern.ch/user/m/masmith/cmtuser/GANGA/GANGA_HEAD/install/ganga/python/GangaDirac/Lib/Utilities/DiracUtilities.py", line 224, in execute
    raise GangaDiracError(returnable['Message'])
GangaDiracError: GangaDiracError: No Output sandbox registered for job 153821230

ERROR    Unable to finalise job after 1466.17 retries due to error:
GangaDiracError: No Output sandbox registered for job 153821240
```
Can we do something better?

**Comment by rob-c** [comment]
This should have been addressed in my PR where I stop the pointless unwanted traces from the queue system.

**Comment by mesmith75** [comment]
Ah ok, great! In which case I'll review that now and lets merge it.

**Comment by mesmith75** [comment]
Fixed by #943
